x-pack/plugins/security_solution/public/plugin_contract.ts

```diff
@@ -21,66 +19,64 @@ export class PluginContract {
   public componentsService: ContractComponentsService;
   public upsellingService: UpsellingService;
   public onboardingPageService: OnboardingPageService;
-  public extraRoutes$: BehaviorSubject<RouteProps[]>;
```

Comment:
We don't have serverless specific routes anymore, all routes are defined in the _security_solution_ plugin